code/modules/surgery/surgery_step.dm

```diff
@@ -67,6 +67,10 @@
 	var/fail_prob = 0//100 - fail_prob = success_prob
 	var/advance = FALSE
 
+	if(surgery.organ_to_manipulate && !target.getorganslot(surgery.organ_to_manipulate))
+		to_chat(user, span_warning("This operation can not be done on [target]!"))
```

Comment:
It would be better if it told the player why the operation can't be done. Something like "[target] lacks the organ needed to complete this operation!" or something.

Reply:
good idea